src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -34,7 +34,10 @@ namespace dawn {
 namespace codegen {
 namespace gt {
 
-GTCodeGen::GTCodeGen(OptimizerContext* context) : CodeGen(context), mplContainerMaxSize_(20) {}
+GTCodeGen::GTCodeGen(std::map<std::string, std::shared_ptr<iir::StencilInstantiation>>& ctx,
+                     DiagnosticsEngine& engine, bool useParallelEP, int maxHaloPoints)
+    : CodeGen(ctx, engine, maxHaloPoints), mplContainerMaxSize_(20), useParallelEP_(useParallelEP) {
```

Comment:
Not sure if needed: For consistency you could move useParallelEP_ into an option struct as in the other generators.

Reply:
we can do that